.travis.yml

```diff
@@ -166,7 +172,9 @@ matrix:
         - sed -n '/``` c++/,/```/{/```/d;p;}' $LITTLEFS/README.md > main.cpp
         - python tools/make.py -t GCC_ARM -m K82F --source=. --build=BUILD/K82F/GCC_ARM -j0
         # Run local littlefs tests
-        - make -C$LITTLEFS/littlefs test QUIET=1
+        - make -C$LITTLEFS/littlefs test QUIET=1 COVERAGE=1
+        # Clean objects which are compiled with gcov
+        - make -C$LITTLEFS/littlefs clean
         # Run local littlefs tests with set of variations
         - make -C$LITTLEFS/littlefs test QUIET=1 CFLAGS+="-DLFS_READ_SIZE=64  -DLFS_PROG_SIZE=64"
```

Comment:
Hmm, should COVERAGE=1 be provided to all of the different test configurations? Is the coverage utility additive?